python/Ganga/__init__.py

```diff
@@ -299,27 +299,13 @@ def getLCGRootPath():
 lcg_config = makeConfig('LCG', 'LCG/gLite/EGEE configuration parameters')
 #gproxy_config = getConfig('GridProxy_Properties')
 
-# set default values for the configuration parameters
-lcg_config.addOption(
-    'EDG_ENABLE', False, 'DEPRECATED enables/disables the support of the EDG middleware')
-
-lcg_config.addOption('EDG_SETUP', '/afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh',
-                 'DEPRECATED sets the LCG-UI environment setup script for the EDG middleware',
-                 filter=Ganga.Utility.Config.expandvars)
-
-lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
```

Comment:
Rather than remove the old config options (I'm not sure Ganga always picks up that options are invalid in a useful way)
Could we not add a method which prevents Ganga starting if the user has configured everything for the old setup to save some confusion about what they have to change.